### PR TITLE
[Enhancement] Supports the agg function hll_raw to speed up the collection of full statistics

### PR DESCRIPTION
**Pull request by HangyuanLiu**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

For TPC-1T lineitem meter, the speed can be increased from the original 10min to 2min30s
## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

File: be/src/exprs/agg/merge_to_hll.h [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+
+namespace starrocks::vectorized {
+
+/**
+ * RETURN_TYPE: TYPE_HLL
+ * ARGS_TYPE: ALL TYPE
+ * SERIALIZED_TYPE: TYPE_VARCHAR
+ */
+template <PrimitiveType PT, typename T = RunTimeCppType<PT>>
+class MergeToHllAggregateFunction final
```
Comment (kangkaisen) [inline be/src/exprs/agg/merge_to_hll.h:20]: could use template to unify the code with HllNdvAggregateFunction

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/FunctionSet.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|1|1|100.00%|[]|

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Review by Seaven — DISMISSED** [review]

**Review by kangkaisen — DISMISSED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by HangyuanLiu** [comment]
run starrocks_be_unittest

**Comment by HangyuanLiu** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:heart_eyes: **pass** : 3 / 3 (100.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/catalog/FunctionSet.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/statistic/FullStatisticsCollectJob.java|1|1|100.00%|[]|

**Pull request merged**